pkg/project/creator.go

```diff
@@ -113,10 +115,15 @@ func (c *Creator) getTemplateValues(values template.Values) (template.Values, er
 }
 
 func (c *Creator) writeFile(targetDir, path string, content []byte, mode os.FileMode) error {
-	log.WithField("path", path).Info("writing file")
-
 	targetPath := filepath.Join(targetDir, path)
 
+	if file.Exists(targetPath) && !c.Options.Overwrite {
+		log.WithField("path", path).Warn("target exists, skipping")
+		return nil
+	}
```

Comment:
I don't like that I need to check that here as well. Might consolidate the logic that writes the optional LICENSE and .gitignore with the logic that writes the actual skeleton files. But this is out of scope of this PR.